service/src/test/java/uk/gov/hmcts/reform/fpl/service/config/LookupTestConfig.java

```diff
@@ -3,8 +3,13 @@
 import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import uk.gov.hmcts.reform.fpl.config.CafcassLookupConfiguration;
+import uk.gov.hmcts.reform.fpl.config.CtscEmailLookupConfiguration;
 import uk.gov.hmcts.reform.fpl.config.HmctsCourtLookupConfiguration;
+import uk.gov.hmcts.reform.fpl.config.LocalAuthorityEmailLookupConfiguration;
 import uk.gov.hmcts.reform.fpl.config.LocalAuthorityNameLookupConfiguration;
+import uk.gov.hmcts.reform.fpl.service.DateFormatterService;
+import uk.gov.hmcts.reform.fpl.service.HearingBookingService;
 
 @Configuration
 public class LookupTestConfig {
```

Comment:
Just a thought about this, probably suited for another PR but it is becoming quite bloated now. What are your thoughts on giving each of these their own. It means that we can then [annotate the classes with the appropriate profiles](https://docs.spring.io/spring/docs/current/spring-framework-reference/core.html#beans-definition-profiles-java) not have to load everything in at once. For example `@Profile("ctscEmail & cafcassLookup")` for a test that requires the `CtscEmailLookupConfiguration` and `CafcassLookupConfiguration`

Reply:
yes, we can do this as a different pr